app/index.js

```diff
@@ -45,237 +62,506 @@ const flow_back = document.getElementById("flow_back");
 const flow_stop = document.getElementById("flow_stop");
 
 // flow buttons json
-const centerButton = document.getElementById("new-button-center");
-const rightButton = document.getElementById("new-button-right");
-const leftButton = document.getElementById("new-button-left");
+let centerButton3 = document.getElementById("new-button-center3");
+let rightButton3 = document.getElementById("new-button-right3");
+let leftButton3 = document.getElementById("new-button-left3");
+
+
+// flow buttons json
+let rightButton2 = document.getElementById("new-button-right2");
+let leftButton2 = document.getElementById("new-button-left2");
+
+
```

Comment:
reduce spacing and reformat code

Reply:
Ok, I'll reformat the code, clean it and add inline documentation in the upcoming request.